Zend/zend_builtin_functions.c

```diff
@@ -1557,27 +1558,66 @@ static void debug_backtrace_get_args(zend_execute_data *call, zval *arg_array) /
 					 */
 					while (i < first_extra_arg) {
 						zend_string *arg_name = call->func->op_array.vars[i];
-						zval *arg = zend_hash_find_ex_ind(call->symbol_table, arg_name, 1);
-						if (arg) {
-							ZVAL_DEREF(arg);
-							Z_TRY_ADDREF_P(arg);
-							ZEND_HASH_FILL_SET(arg);
+						zval arg;
+						zval *old_arg = zend_hash_find_ex_ind(call->symbol_table, arg_name, 1);
```

Comment:
Nit: `unredacted_arg`? `old_` makes it sound like the previous arg. Sorry I didn't mention that in the previous review.

Reply:
> Sorry I didn't mention that in the previous review.

No problem. I've opted for s/old/original/ and s/new/redacted/: https://github.com/php/php-src/compare/405914a9b8ccd27f3a2788fe61f39f68ec0c3c25..846a52d5db0541679774ebbf5de1dcf70b8a1371

Reply:
I've also just swapped "arg" and "original_arg", as that naming makes a little more sense then with the scoping.